recipes/connectorx/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "connectorx" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/sfu-db/connector-x/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: c8d13ac38d9da5793f8ccb0c3777761bcfaa89f0f18cbf0597d0bd4f347612c9
+
+build:
+  number: 0
+  skip: true  # [py<37 or win]
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - crossenv                            # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - posix                               # [win]
+    - cmake
+    - make                                # [unix]
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
+    - poetry
```

Comment:
Poetry is a different package manager. Is it possible to use only conda packages and remove poetry from the required list of packages?